[bug]: Fix failing LIly tests

### What seems to be the problem? 🤔

I've merged in Lily from #6212 because I did not want it to fall behind, but it needs some work.
16 of its unit tests fail, which need to be addressed. Assining to @anna-puk 

### Design / Plugin / Package 🧐

_No response_

### Additional context

```
  1) Pattern translation:
         - The design should have a translated title:

      AssertionError: expected 4 to be above 4
      + expected - actual


      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:17:38)
      at process.processImmediate (node:internal/timers:476:21)

  2) Pattern translation:
         - The option title for the "fitGuides" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  3) Pattern translation:
         - The option title for the "waistEase" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  4) Pattern translation:
         - The option title for the "seatEase" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  5) Pattern translation:
         - The option title for the "kneeEase" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  6) Pattern translation:
         - The option title for the "waistHeight" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  7) Pattern translation:
         - The option title for the "crotchDrop" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  8) Pattern translation:
         - The option title for the "fitKnee" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  9) Pattern translation:
         - The option title for the "crossSeamCurveStart" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  10) Pattern translation:
         - The option title for the "crossSeamCurveBend" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  11) Pattern translation:
         - The option title for the "crossSeamCurveAngle" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  12) Pattern translation:
         - The option title for the "crotchSeamCurveStart" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  13) Pattern translation:
         - The option title for the "crotchSeamCurveBend" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  14) Pattern translation:
         - The option title for the "crotchSeamCurveAngle" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  15) Pattern translation:
         - The option title for the "grainlinePosition" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  16) Pattern translation:
         - The option title for the "waistbandWidth" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)


```

--- a/designs/lily/i18n/en.json
+++ b/designs/lily/i18n/en.json
@@ -1,5 +1,5 @@
 {
-  "t": "Lily",
+  "t": "Lily Leggings",
   "d": "A FreeSewing pattern for basic leggings",
   "p": {
     "back": "Back",
@@ -19,6 +19,66 @@
     "fabricStretch": {
       "t": "Fabric stretch",
       "d": "How much the fabric can stretch horizontally; this is used to calculate ease"
+    },
+    "fitGuides": {
+      "t": "Show measurement guides",
+      "d": "Show measurement guides for horizontal measures"
+    },
+    "waistEase": {
+      "t": "Waist ease",
+      "d": "Controls the amount of ease at your waist"
+    },
+    "seatEase": {
+      "t": "Seat ease",
+      "d": "Controls the amount of ease at your seat"
+    },
+    "kneeEase": {
+      "t": "Knee ease",
+      "d": "Controls the amount of ease at the knee"
+    },
+    "waistHeight": {
+      "t": "Waist height",
+      "d": "Controls the height of the waist, 100% = waist height, 0% = hip height"
+    },
+    "crotchDrop": {
+      "t": "Crotch drop",
+      "d": "Lowers the crotch for a more relaxed fit"
+    },
+    "fitKnee": {
+      "t": "Fit the knee",
+      "d": "Fits the legs from based on the knee circumference, rather than seat circumference"
+    },
+    "crossSeamCurveStart": {
+      "t": "Start of the cross seam curve",
+      "d": "Controls how far into the cross seam we start to curve"
+    },
+    "crossSeamCurveBend": {
+      "t": "Cross seam bend",
+      "d": "Controls the curvature of the cross seam"
+    },
+    "crossSeamCurveAngle": {
+      "t": "Cross seam angle",
+      "d": "Controls the angle of the cross seam"
+    },
+    "crotchSeamCurveStart": {
+      "t": "Start of the crotch seam curve",
+      "d": "Controls how far into the crotch seam we start to curve"
+    },
+    "crotchSeamCurveBend": {
+      "t": "Crotch seam bend",
+      "d": "Controls the curvature of the crotch seam"
+    },
+    "crotchSeamCurveAngle": {
+      "t": "Crotch seam angle",
+      "d": "Controls the angle of the crotch seam"
+    },
+    "grainlinePosition": {
+      "t": "Grainline position",
+      "d": "Controls the horizontal position of the leg relative to the seat"
+    },
+    "waistbandWidth": {
+      "t": "Waistband width",
+      "d": "The width of the waistband"
     }
   }
 }

